packages/slice-machine/components/Simulator/index.tsx

```diff
@@ -80,7 +80,7 @@ export default function Simulator() {
       />
       <ScreenshotPreviewModal
```

Comment:
OPT: I think it would make more sense to display the modal only if there is an actual screenshot.